akamai/resource_akamai_fastdns_zone.go

```diff
@@ -1287,3 +1288,26 @@ func resourceFastDNSZoneExists(d *schema.ResourceData, meta interface{}) (bool,
 	zone, err := dns.GetZone(hostname)
 	return zone != nil, err
 }
+
+func saveZone(zone *dns.Zone) error {
+	log.Printf("[DEBUG] [Akamai FastDNS] Saving zone")
+	if zone.Zone.Soa != nil {
+		zone.Zone.Soa.Serial += 1
```

Comment:
Why is this necessary now? did `zone.Save()` used to do it automatically?

Reply:
Yes, see commit message in e03fc87372cfd4494d332b759b6cb933942144cf. Newer versions only increment the serial number if it hasn't changed, but the field containing the original serial number is not included in the map produced by `ToMap`. So, if the SOA is converted to and from a map, it appears that the serial number _has_ changed (`originalSerial` is now a zero value), and it's not incremented.